code-challenges.test.js

```diff
@@ -73,8 +87,25 @@ it("takes an array of numbers and returns an array of the minimum and maximum nu
     expect(minMaxnum(nums1)).toEqual([-8, 90])
     expect(minMaxnum(nums2)).toEqual([5, 109])
   })
-})
+//  FAIL  ./code-challenges.test.js
+//   ● Test suite failed to run
+// FAIL  ./code-challenges.test.js
+//   minMaxnum
+//     ✕ matches even if received contains additional elements (1 ms)
+
+//   ● minMaxnum › matches even if received contains additional elements
+
+//     ReferenceError: minMaxnum is not defined})
 
+
+
+
+
+//1.make function 
+//2.I wanted to sort array from numerically
+//3.I wanted to take the first index of array and save in a variable
+//4.I wanted to save the last variable into a variable 
+//5. I tried to create a new variable that held the first 2 variables in concatted form.
 //
 const minMaxnum =(arr)=>{
     arr.sort()
```

Comment:
I see 2 different approaches here.  .pop() and .shift() would remove the elements that you would want to utilize.  If you use .sort(), you will need to add some more information to sort numbers correctly: `arr.sort((a, b) => a - b)`.  You will also need to get your first value and last value in the array and place them into another array.  This can be a done in a number of ways.  One by making an array with the values inside it such as: `[arr[0], arr[arr.length-1]]`.